addon/templates/components/ember-attacher.hbs

```diff
@@ -10,7 +10,7 @@
                 target=target as |emberPopper|}}
   {{#ember-attacher-inner animation=animation
                           arrow=arrow
-                          class="inner"
+                          class=(concat 'inner' ' ' innerClass)
```

Comment:
can't this just be `(concat 'inner ' innerClass)`?

Reply:
Yep! Changing now